backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -509,17 +504,18 @@ export class ExperimentAssignmentService {
       }, []);
     } catch (err) {
       const error = err as ErrorWithType;
-      this.log.error('Error in assignment');
+      logger.error({ stdout: 'Error in assignment', stack_trace: error});
```

Comment:
stack_trace should be `error.stack`.
I think you should log the whole error object here and if you need to add more detail you can add that in error object itself.
`err.detail = 'Error in assignment'`

Reply:
done